weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/CallsTableButtons.tsx

```diff
@@ -90,6 +103,16 @@ export const ExportSelector = ({
       ? visibleColumns
       : undefined;
     const startTime = Date.now();
+
+    visibleColumns.sort((a, b) => b.length - a.length);
+    const minimalColumns: string[] = [];
+    for (const col of visibleColumns) {
+      if (minimalColumns.some(minimalCol => minimalCol.startsWith(col))) {
```

Comment:
Why are you calculating "minimalColumns"? This seems incorrect. If my visible columns are: `a` and `a.b`, why do i only send over `a.b`?

Reply:
I see, so `columns` should be the "leaf columns"